autogluon/task/tabular_prediction/tabular_prediction.py

```diff
@@ -129,12 +129,16 @@ def fit(train_data, label, tuning_data=None, output_directory=None, problem_type
                      and the lightGBM docs: https://lightgbm.readthedocs.io/en/latest/Parameters.html
                 CAT: See `file autogluon/utils/tabular/ml/models/catboost/hyperparameters/parameters.py` 
                      and the CatBoost docs: https://catboost.ai/docs/concepts/parameter-tuning.html
-                RF: `n_estimators` is currently the only hyperparameter you can specify, 
-                    see sklearn docs: https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestClassifier.html 
-                XT: `n_estimators` is currently the only hyperparameter you can specify, 
-                    see sklearn docs: https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.ExtraTreesClassifier.html
-                KNN: Currently no hyperparameters may be specified for k-nearest-neighbors models
-                
+                RF: See sklearn docs: https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestClassifier.html
+                    Note: Hyperparameter tuning is disabled for this model.
+                    Note: 'criterion' parameter will be overriden.
+                XT: See sklearn docs: https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.ExtraTreesClassifier.html
+                    Note: Hyperparameter tuning is disabled for this model.
+                    Note: 'criterion' parameter will be overriden.
```

Comment:
should clarify: 'criterion' parameter will be overridden by what?

Reply:
Added